gravitee-apim-console-webui/src/management/api/portal/subscriptions/components/dialogs/creation/api-portal-subscription-creation-dialog.component.spec.ts

```diff
@@ -140,16 +144,19 @@ describe('Subscription creation dialog', () => {
 
         const harness = await loader.getHarness(ApiPortalSubscriptionCreationDialogHarness);
         expect(await harness.isCustomApiKeyInputDisplayed()).toBeFalsy();
-
-        await harness.choosePlan(planV4.name);
+        expect(await harness.isPlanRadioGroupDisabled()).toBeTruthy();
```

Comment:
I have just a personal preference for the other logic.
I would have written `isPlanRadioGroupEnabled` and expect it to be "falsy", just to follow the logic of the other input

Reply:
I wanted to be closer to HTML `disabled`

Reply:
as discussed, I use your solution for the consistency